packages/SwingSet/test/virtualObjects/test-representatives.js

```diff
@@ -387,7 +387,7 @@ test('virtual object gc', async t => {
     [`${v}.vs.vc.4.|entryCount`]: '0',
     [`${v}.vs.vc.4.|label`]: 'watchedPromises',
     [`${v}.vs.vc.4.|nextOrdinal`]: '1',
-    [`${v}.vs.vc.4.|schemata`]: vstr([M.string()]),
+    [`${v}.vs.vc.4.|schemata`]: vstr([M.and(M.scalar(), M.string())]),
```

Comment:
@FUDCo I "fixed" this test so that it passes, without really understanding what's going on. Did I change the test correctly?

Reply:
This seems weird and wrong.  The table in question, created by a call to `makeScalarBigMapStore`,  has keys that are strictly strings and is declared as such in `swingset-liveslots/src/watchedPromises.js` (which is not one of the files changed in this PR).  I don't where the "AND scalar" part is creeping in, which suggests maybe something is broken here.

Reply:
Discussion with @FUDCo and @warner figured out the result above. The one with the `M.and(...)` is going through the normal scalarizing wrapper. The that just says `M.string()`  is using collectionManager's `provideBaggage` with bypasses the wrappers, since it is inside the module and already known that string implies scalar.

So resolving this conversation.